code/mob/living/critter/small_animal.dm

```diff
@@ -1957,6 +1957,133 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 			reduce_lifeprocess_on_death()
 			..()
 
+/* ================================================ */
+/* ------------------- Iguana --------------------- */
+/* ================================================ */
+
+/mob/living/critter/small_animal/iguana
+	name = "space iguana"
+	real_name = "space iguana"
+	desc = "An iguana that came from space. Or maybe went to space. Who knows how it got here?"
+	icon_state = "iguana1"
+	icon_state_dead = "iguana1-dead"
+	hand_count = 2
+	speechverb_say = "hisses"
+	speechverb_exclaim = "wheezes"
+	butcherable = 0
+	health_brute = 15
+	health_burn = 15
+	pet_text = list("gently baps", "pets", "cuddles")
+	density = 1
+	fits_under_table = TRUE
+	var/on_tree
+
+	New()
+		. = ..()
+		START_TRACKING
+		if(prob(20))
+			icon_state = "iguana2"
+			icon_state_dead = "iguana2-dead"
+
+	disposing()
+		. = ..()
+		STOP_TRACKING
```

Comment:
Probably not an issue here but it's a good idea to call disposing parent last. As this sort of a closing brack to New where its the other way around, I'm sure you get what I mean.
```suggestion
		STOP_TRACKING
		. = ..()
```